build.gradle

```diff
@@ -115,32 +115,32 @@ dependencies {
     implementation "com.google.cloud.opentelemetry:exporter-metrics:${gcpOpenTelemetryExporterVersion}"
 
     // these are required for tracing in tests
-    testImplementation 'org.springframework:spring-aop:6.0.11'
-    testImplementation 'org.springframework:spring-aspects:6.0.11'
+    testImplementation 'org.springframework:spring-aop:6.1.3'
+    testImplementation 'org.springframework:spring-aspects:6.1.3'
 
 
     // Spotbugs
-    implementation group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: '4.7.2'
+    implementation group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: '4.8.3'
```

Comment:
I have an open PR to remove spotbugs, so I'm waiting to rebase this one before moving it out of draft.

Reply:
Removed spotbugs-annotations from the PR.